src/script/conversation/ConversationRepository.test.ts

```diff
@@ -100,6 +100,8 @@ describe('ConversationRepository', () => {
   };
 
   beforeAll(async () => {
+    jest.useFakeTimers();
```

Comment:
Is this somehow breaking some other test cases? The ci is breaking

Reply:
maybe, i'm checking...

Reply:
https://github.com/wireapp/wire-webapp/pull/15158/commits/5459a623a56bd63ccda7c72b8d3e168a32575835